catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java

```diff
@@ -170,10 +169,21 @@ public UpdateResponse update(UpdateRequest updateRequest)
         } catch (StopProcessingException see) {
             throw new IngestException(PRE_INGEST_ERROR, see);
         } catch (RuntimeException re) {
-            throw new InternalIngestException("Exception during runtime while performing update", re);
+            throw new InternalIngestException("Exception during runtime while performing update",
+                    re);
         }
     }
 
+    @SuppressWarnings("unchecked")
+    private Map<String, Metacard> getUpdateMap(UpdateRequest updateRequest) {
+        return (Map<String, Metacard>) Optional.of(updateRequest)
+                .map(Operation::getProperties)
+                .map(p -> p.get(Constants.ATTRIBUTE_UPDATE_MAP_KEY))
+                .filter(m -> m instanceof Map)
```

Comment:
`Map.class::isInstance` should be functionally equivalent.
From [the javadocs](https://docs.oracle.com/javase/8/docs/api/java/lang/Class.html#isInstance-java.lang.Object-):
> This method is the dynamic equivalent of the Java language instanceof operator. The method returns true if the specified Object argument is non-null and can be cast to the reference type represented by this Class object without raising a ClassCastException. It returns false otherwise.